### PR TITLE
[Sparse Summit - Meeting 1] Add link to scikit-learn BoF presentation

### PR DESCRIPTION
**Pull request by jjerphan**
Hi @rossbar and @jarrodmillman.

Here is the link to my presentation for Sparse Summit - Meeting 1 on next Monday.

One note: I slightly have adapted the title of this presentation as I do not know what "roles" stands for in this context.

Please, let me know if everything is as intended. :slightly_smiling_face:

### CODE REVIEW

File: content/summits/sparse/meeting1.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ birds-of-a-feather (BoF)-style talks, followed by more focused discussion.
    - Hameer Abbasi: Pydata-sparse current and future work
    - Jim Kitchen: needs/roles of sparse in GraphBLAS
    - Isaac Virshup: needs/roles of sparse in scverse
-   - Julien Jerphanion: needs/roles of sparse in `scikit-learn`
+   - [Julien Jerphanion: usage and needs for sparse data in `scikit-learn`](https://docs.google.com/presentation/d/1wsl03LXpmmhlKKkcnKUToCq0xaaLPyAmXP9vypjvsfg/edit?usp=sharing)
```
Comment (tupui) [inline content/summits/sparse/meeting1.md:35]: I think it would be better to self host such things. It's hard to keep track of external links.

Comment (jjerphan) [reply content/summits/sparse/meeting1.md:35]: I agree. Where would you see it self-hosted?

Comment (tupui) [reply content/summits/sparse/meeting1.md:35]: We could have it in this repo or another one. This is more a question for @stefanv and @jarrodmillman 😃

Comment (jarrodmillman) [reply content/summits/sparse/meeting1.md:35]: I will set up something for this soon.

Comment (jjerphan) [reply content/summits/sparse/meeting1.md:35]: Sorry for this interjection, @jarrodmillman, but it https://github.com/scientific-python/scientific-python.org-blobs meant to store blobs of such presentations as well?

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**